### ISSUE TITLE
Update reference to IDPF

### ISSUE THREAD

**Issue by ways2read** [daisy/epub-accessibility-tests#9]
In the next release of the a11y test books, the reference to IDPF should be updated.
"This publication is considered stable by the IDPF and DAISY Consortium"

**Comment by avneeshsingh** [comment]
refer to @georgekerscher.
